### PR TITLE
Repository cleanup

### PR DESCRIPTION
**Pull request by aboucaud**
- few dependency fixes after testing the installation
- add setup paragraph
- .editorconfig

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```
Comment (EiffL) [inline .editorconfig:8]: actually ^^ I like indent_size=2

**Review by EiffL — APPROVED** [review]
Very nice!

File: .editorconfig (hunk 1 above)
Comment (aboucaud) [reply .editorconfig:8]: I like even more consistency (and `setup.py` was 4 spaces indented :-P)

Comment (aboucaud) [reply .editorconfig:8]: But ok, let's try indent_size=2 for this repo..

**Pull request merged**